### ISSUE TITLE
Graphical admin panel

### ISSUE THREAD

**Issue by p2r3** [p2r3/epochtal#9]
Implement an admin panel for simpler management of the tournament. No changes need to be made to the backend, as existing utils are more than enough.

**Comment by ArcherTheReal** [comment]
This only needs to do utility calls right?

**Comment by p2r3** [comment]
@ArcherTheReal Right. There's nothing you'd need for the admin panel that you can't get from `util` calls. Since it seems like you're looking into potentially implementing this, let me give you a rundown of how I envision this looking:

The admin panel would be located under `pages/admin/index.html` and would use the default `pages/style.css` styling. Ideally, existing classes would be used for laying out the page instead of custom styling specifically for the admin panel. The following features should be included:
- Fields for modifying the week's configuration (week number, picked map, screenshot url, etc.)
- Totals for map votes, showing cumulative up/down votes for each map
- Category list with options to edit their properties (name, title, flags, order, etc.)
- Category creation form
- Buttons that run routines (and probably pop out the CLI to monitor UtilPrint)
- Shortcut for locking all categories (and potentially other useful shortcuts)
